views/index.pug

```diff
@@ -21,7 +21,19 @@ block content
         may not work correctly but are provided for your convenience. <strong>Do not share this zip file with others!</strong>
         The mod bundle includes credentials that identify which streamer page to update. If another user installs the same
         zip bundle, they will be sending wand updates to <em>your</em> stream page!
-
+      h2.progress Progress Instructions
+      p.
+        Spells and Perks progress is fully tracked by streamer-wands and requires no additional steps by you.
+        Enemy progress is tracked assuming you are on a clean save file (AKA no enemy progress), if you want 
+        tracking to start from your current enemy progress follow the below steps:<br><br>
+        1. Browse to your <strong>%appdata%/LocalLow/Nolla_Games_Noita/save00/stats/_stats.salakieli</strong><br>
+        2. Download the stats.lua file<br>
+        3. Move the stats.lua inside streamer-wands, root directory so right next to token.lua and mod.xml<br>
```

Comment:
I'm going to have to spend a little time on code review, probably will roll it into a stream with the final testing of NT delayed graceful shutdown. I skimmed the changes, and from a user POV this stood out to me as a minor improvement - the instructions here don't explicitly tell the user what to do. It says "browse to", but not in relation to "select the file using the input below". Probably something like this would be a simple but meaningful help:

```suggestion
        1. Click "browse" below
        1. Browse to your <strong>%appdata%/LocalLow/Nolla_Games_Noita/save00/stats/_stats.salakieli</strong><br>
        1. Download the stats.lua file<br>
        1. Move the stats.lua inside streamer-wands, root directory so right next to token.lua and mod.xml<br>
```